blocks/init/src/Blocks/custom/tabs/tabs-editor.scss

```diff
@@ -0,0 +1,18 @@
+.block-tabs {
+	&__panelitem {
+		padding: 0.5em;
```

Comment:
We're still on `px` for all other components if I'm not mistaken. While this might be better / more correct, I'd rather if we have consistency across all our components.

Then we can update all components at one point to use em / rem values.

Reply:
Some components use (r)em: 

- [accordion](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/components/accordion/accordion-style.scss#L2)
- [card](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/components/card/card-style.scss#L5)
- [button](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/components/button/button-style.scss#L10)
- [jumbotron](https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/components/jumbotron/jumbotron-style.scss#L17)
- (perhaps others as well)

Reply:
we are removing px all together.  Please check other blocks/components

Reply:
Will remove the `margin-bottom` property entirely as using `overrideInnerBlockSimpleWrapperAttributes` mitigates the need to modify this margin.